### ISSUE TITLE
413 Request Entity Too Large for typical 16 GB archive

### ISSUE THREAD

**Issue by artemiy-shkurdin** [codl/forget#262]
Getting 413 Request Entity Too Large for typical 16 GB archive
Need a way to upload larger files

**Comment by codl** [comment]
Typical? How many posts is that? Are attempting to upload a "tweet archive" (just tweets) or "your twitter data" (tweets and media and other things)? If you don't have javascript globally disabled, you should have been shown a warning message before you tried to upload a file that big.

See also https://github.com/codl/forget/issues/119#issuecomment-435915403

**Comment by artemiy-shkurdin** [comment]
Well then, if there is a way to extract only the data Forget needs, then why won't just add a detailed guide in case of such errors? Or even preemptively reference it at the main part of a page

**Comment by codl** [comment]
No, the format between the two kinds of archive is very different and it's far easier to just download the other kind of archive instead

**Comment by codl** [comment]
It might be interesting to have the 413 error page mention the difference between both archives and which one is correct. I'm not sure that my web server software will let me do that.
